site/src/components/Dialogs/DeleteDialog/DeleteDialog.tsx

```diff
@@ -1,31 +1,80 @@
-import React, { ReactNode } from "react"
+import FormHelperText from "@material-ui/core/FormHelperText"
+import makeStyles from "@material-ui/core/styles/makeStyles"
+import TextField from "@material-ui/core/TextField"
+import Typography from "@material-ui/core/Typography"
+import { Maybe } from "components/Conditionals/Maybe"
+import { Stack } from "components/Stack/Stack"
+import React, { ChangeEvent, useState } from "react"
+import { useTranslation } from "react-i18next"
 import { ConfirmDialog } from "../ConfirmDialog/ConfirmDialog"
 
 export interface DeleteDialogProps {
   isOpen: boolean
   onConfirm: () => void
   onCancel: () => void
-  title: string
-  description: string | ReactNode
+  entity: string
+  name: string
+  info?: string
   confirmLoading?: boolean
 }
 
 export const DeleteDialog: React.FC<React.PropsWithChildren<DeleteDialogProps>> = ({
   isOpen,
   onCancel,
   onConfirm,
-  title,
-  description,
+  entity,
+  info,
```

Comment:
What kind of info is this? A description?

Reply:
Yeah, it's intended to help the user identify the entity to delete, so they don't delete the wrong one by accident. @bpmct suggested this on the basis of an article about making the wrong repo private and thus losing all their github stars.

Reply:
It's the orange text here. Note storybook isn't showing buttons; that's been going on for a while and so far doesn't mean the buttons are actually gone. 
<img width="615" alt="Screen Shot 2022-09-20 at 4 52 28 PM" src="https://user-images.githubusercontent.com/1290996/191362220-31283007-42c2-4d98-b8ba-1264c43e0104.png">